package.json

```diff
@@ -61,12 +62,6 @@
         {
           "pkgRoot": "packages/openapi-types-ghes-3.5"
         }
-      ],
-      [
-        "@semantic-release/npm",
-        {
-          "pkgRoot": "packages/openapi-types-github.ae"
-        }
```

Comment:
Is there a reason for removing this?

Reply:
Nope! This was caused by stale changes in my branch. I'll fix that.